tests/CMakeLists.txt

```diff
@@ -12,23 +12,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set(CMAKE_FOLDER "${CMAKE_FOLDER}/tests")
```

Comment:
Why using this instead of 
`set_target_properties(tests_util PROPERTIES FOLDER "VulkanLayerUtils-tests")`
inc?

I thought a rule was to not manually modify CMake variable if we don't have to?

Reply:
If more tests get added it's nice to automatically have all the tests under the `tests/` filter. Instead of having to add it one by one. It's similar to CMAKE_POSITION_INDEPENDENT_CODE.